src/preact/contextprops.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {Loading, reducer as loadingReducer} from '../loading';
```

Comment:
I think these actually belong in `src/core`? They're meant to be used by both Bento and regular Components?
/cc @dvoytenko

Reply:
AFAICT is file is not imported anywhere else in the repo except the files in this PR